tests/python/pants_test/engine/legacy/test_console_rule_integration.py

```diff
@@ -13,29 +13,19 @@
 
 class TestConsoleRuleIntegration(PantsDaemonIntegrationTestBase):
 
+  list_target = 'examples/src/java/org/pantsbuild/example/hello::'
+
   @ensure_daemon
   def test_v2_list(self):
-    result = self.do_command(
-      '--no-v1',
-      '--v2',
-      'list',
-      '::',
```

Comment:
We no longer glob against everything. This is not feasible because we cannot sanely depend on every single file in the repo.